src/common/sys_lmdb.c

```diff
@@ -128,6 +129,13 @@ lmm_db_open_create(struct sys_db *db, bool try_create)
 		goto out;
 	}
 
+	rc = mdb_env_set_mapsize(ldb->db_env, SYS_DB_MAX_MAP_SIZE);
```

Comment:
I think we'd better understand the issue before bumping the map size. I can't image how SMD could use up 10MB map size.

Reply:
It might be a typo bug of manual, checking codes, it looks it was always 1MiB...
[root@daos1 tmp]# grep -R MAPSIZE lmdb-LMDB_0.9.24/
lmdb-LMDB_0.9.24/libraries/liblmdb/mdb.c:#define DEFAULT_MAPSIZE        1048576
lmdb-LMDB_0.9.24/libraries/liblmdb/mdb.c:               meta.mm_mapsize = DEFAULT_MAPSIZE;


[root@daos1 lmdb]# git grep DEFAULT_MAPSIZE
libraries/liblmdb/mdb.c:#define DEFAULT_MAPSIZE 1048576
libraries/liblmdb/mdb.c:                meta.mm_mapsize = DEFAULT_MAPSIZE;